### PR TITLE
add some class level comments to the files in apps/

### PR DESCRIPTION
**Pull request by johrstrom**
Add some class level comments to the files in `apps/`.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1202956058864971) by [Unito](https://www.unito.io)

### CODE REVIEW

File: apps/dashboard/app/apps/path_router.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # A special router to use to instantiate an OodApp
-# object if all you have is the path to the app
+# object if all you have is the path to the app.
+#
+# Not used at runtime in any environment except for test.
+# This is a helper class for unit testing when SysRouter or similar is to heavy.
```
Comment (Oglopf) [inline apps/dashboard/app/apps/path_router.rb:5]:
```suggestion
# This is a helper class for unit testing when SysRouter or similar is too heavy.
```

File: apps/dashboard/app/apps/router.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Generic Router class (as a pose to SysRouter that is specific to system apps)
```
Comment (Oglopf) [inline apps/dashboard/app/apps/router.rb:1]:
```suggestion
# Generic Router class (as apposed to SysRouter that is specific to system apps)
```
Not sure if this is maybe what you meant?

Comment (johrstrom) [reply apps/dashboard/app/apps/router.rb:1]: LOL yes.

File: apps/dashboard/app/apps/manifest.rb [hunk 3]
```diff
@@ -52,15 +55,15 @@ def self.load_from_string(yaml)
...
+  # @option opts [String] :name The name of the application.
+  # @option opts [String] :description The description of the application.
+  # @option opts [String] :category The category of the application.
+  # @option opts [String] :subcategory The subcategory of the application.
+  # @option opts [String] :icon The icon used on the dashboard, optionally a Font Awesome tag.
+  # @option opts [String] :role Dashboard categorization.
+  # @option opts [String] :url An optional redirect URL.
+  # @option opts [Hash]   :metadata An optional hash of key value pairs.
   def initialize(opts)
     raise InvalidContentError.new unless(opts && opts.respond_to?(:to_h))
```
Comment (gerald-byrket) [inline apps/dashboard/app/apps/manifest.rb:69]: This comment is very helpful. Thank you for these details.

**Review by gerald-byrket — APPROVED** [review]

**Pull request merged**